elasticdl/python/master/evaluation_service.py

```diff
@@ -13,12 +12,48 @@
 class _EvaluationJob(object):
     """Representation of an evaluation job"""
 
-    def __init__(self, model_version, total_tasks=-1):
+    def __init__(self, metrics_dict, model_version, total_tasks=-1):
+        """
+        Args:
+            metrics_dict: A python dictionary. If model has only one output,
```

Comment:
Specify that each metric uses only one model output.

Reply:
Done.